content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -180,7 +201,7 @@ getting replaced.
 ## {{% heading "whatsnext" %}}
 
 
-* [Enabling EndpointSlices](/docs/tasks/administer-cluster/enabling-endpointslices)
+* [Enabling Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpoint-slices)
```

Comment:
Hello @robscott .
nit: Check this link. Update to: /docs/tasks/administer-cluster/enabling-endpointslices

Reply:
Woops, not sure how I mixed that up, thanks!